conda_build/os_utils/pyldd.py

```diff
@@ -1045,6 +1052,15 @@ def inspect_rpaths(filename, resolve_dirnames=True, use_os_varnames=True,
                 return cf.rpaths_nontransitive
 
 
+def get_runpaths(filename, arch='native'):
+    if not os.path.exists(filename):
+        return []
+    arch = _get_arch_if_native(arch)
+    with open(filename, 'rb') as f:
+        cf = codefile(f, arch, ['/lib', '/usr/lib'])
```

Comment:
What about /lib64 , /usr/lib64 ?

Reply:
That may need a bit of a cleanup after I remind myself why I didn't do that, but the same thing happens in the existing, well-trodden codepaths and it's harmless there at least (even if I cannot completely explain it at present!)

All we actually care about for now is whether any DT_RUNPATHs are present and not their resolved (or unresolved) location.